src/main/groovy/life/qbic/datamodel/dtos/imaging/Image.groovy

```diff
@@ -1,12 +1,125 @@
 package life.qbic.datamodel.dtos.imaging
 
+import life.qbic.datamodel.dtos.imaging.parameters.AcquisitionParameters
+import life.qbic.datamodel.dtos.imaging.parameters.AnalysisParameters
+import life.qbic.datamodel.dtos.imaging.properties.ImageSize
+import life.qbic.datamodel.dtos.imaging.properties.Modality
+import life.qbic.datamodel.dtos.imaging.properties.PixelSize
+import life.qbic.datamodel.dtos.imaging.properties.Resolution
+
 /**
- * <class short description - 1 Line!>
+ * This DTO class describes an image resulted from imaging analysis
  *
- * <More detailed description - When to use, what it solves, etc.>
+ * An image class shall be used to describe properties of an image
+ * in the context of imaging analysis such as microscopy or tomography.
  *
  * @author Sven Fillinger
- * @since <versiontag>
+ * @since 1.10.0
  */
-class Image {
+final class Image {
+  /**
+   * The image acquisition date.
+   *
+   * @see Date
+   */
+  private final Date acquisitionDate
+
+  /**
+   * Detailed description of the image acquisition parameters.
+   *
+   * @see AcquisitionParameters
+   */
+  private final AcquisitionParameters acquisitionParameters
+
+  /**
+   *
+   */
+  private final AnalysisParameters analysisParameters
+
+  /**
+   * A detailed instrument description that was used to create the image.
+   */
+  private final Instrument instrument
+
+  /**
+   * The image identifier in OMERO.
+   */
+  private final Integer imageId
```

Comment:
Do we want to make the name more explicit? Maybe something like `omeroImageId`?
Otherwise it might get confused with IDs in openBis.

Reply:
Good point. Do you want to suggest the name change to get contribution?

Reply:
ah, you already get as I will squash the commi :) I will adjust the name

Reply:
Hm, now that I have thought about it, the GroovyDoc String already mentioned it. Do you think this is enough?

Reply:
I am not sure.. this will not directly be seen if you only see the variable. Maybe we need to ask the others?